src/SM/Factory/FactoryInterface.php

```diff
@@ -24,4 +24,9 @@
      * @return StateMachineInterface
      */
     public function get($object, $graph = 'default');
+    
+    /**
+     * Clears all state machines from the factory
+     */
+    public function clear();
```

Comment:
Can you put it in a different interface? Like `ClearableFactoryInterface` that would extend the default interface? I'd like to avoid BC break as not everyone uses the default factory. Thanks!